libcontainer/apparmor/apparmor.go

```diff
@@ -12,11 +13,9 @@ import (
 
 // IsEnabled returns true if apparmor is enabled for the host.
 func IsEnabled() bool {
-	if _, err := os.Stat("/sys/kernel/security/apparmor"); err == nil && os.Getenv("container") == "" {
-		if _, err = os.Stat("/sbin/apparmor_parser"); err == nil {
-			buf, err := ioutil.ReadFile("/sys/module/apparmor/parameters/enabled")
-			return err == nil && len(buf) > 1 && buf[0] == 'Y'
-		}
+	if _, err := os.Stat("/sys/kernel/security/apparmor"); err == nil {
+		buf, err := ioutil.ReadFile("/sys/module/apparmor/parameters/enabled")
```

Comment:
No need to call `os.Stat()` here as we already check for error from `ioutils.ReadFile`.

So this should be something like
```go
func IsEnabled() bool {
   buf, err := ioutil.ReadFile("/sys/module/apparmor/parameters/enabled")
   return err == nil && bytes.HasPrefix(buf, []byte("Y"))
}
```

Reply:
No, the `Stat` is necessary -- the paths are slightly different (one is for `apparmorfs` and the other is the module parameter check).

Reply:
Ah, I overlooked it, thanks!